ServerRecording/Controller/CallRecordingController.cs

```diff
@@ -0,0 +1,352 @@
+        /// <param name="request"></param>
+        /// <returns></returns>
+        [HttpPost]
+        [Route("getRecordingFile")]
+        public async Task<ActionResult> GetRecordingFile([FromBody] object request)
+        {
+            try
+            {
+                //Deserializing the request 
+                var eventGridEvent = JsonConvert.DeserializeObject<EventGridEvent[]>(request.ToString()).FirstOrDefault();
+
+                if (string.Equals(eventGridEvent.EventType, EventTypes.EventGridSubscriptionValidationEvent, StringComparison.OrdinalIgnoreCase))
+                {
+                    var data = eventGridEvent.Data as JObject;
+                    var eventData = data.ToObject<SubscriptionValidationEventData>();
+
+                    Logger.LogInformation("Microsoft.EventGrid.SubscriptionValidationEvent response  -- >" + eventGridEvent.Data);
+
+                    var responseData = new SubscriptionValidationResponse
+                    {
+                        ValidationResponse = eventData.ValidationCode
+                    };
+
+                    if (responseData.ValidationResponse != null)
+                    {
+                        return Ok(responseData);
+                    }
+                }
+
+                if (string.Equals(eventGridEvent.EventType, "Microsoft.Communication.RecordingFileStatusUpdated", StringComparison.OrdinalIgnoreCase))
+                {
+                    Logger.LogInformation($"Event type is -- > {eventGridEvent.EventType}");
+
+                    Logger.LogInformation("Microsoft.Communication.RecordingFileStatusUpdated response  -- >" + eventGridEvent.Data);
+
+                    RecordingFileData data = EventSerializer.DeserializeObject<RecordingFileData>(eventGridEvent.Data.ToString());
+
+                    Logger.LogInformation("Start processing recorded media -- >");
+
+                    await ProcessFile(data.recordingStorageInfo.recordingChunks[0].contentLocation,
+                        data.recordingStorageInfo.recordingChunks[0].documentId,
+                        "mp4",
+                        "recording");
+
+                    Logger.LogInformation("Start processing metadata -- >");
+
+                    await ProcessFile(data.recordingStorageInfo.recordingChunks[0].metadataLocation,
+                        data.recordingStorageInfo.recordingChunks[0].documentId,
+                        "json",
+                        "metadata");
+                }
+
+                return Ok();
+            }
+            catch (Exception ex)
+            {
+                return Json(new { Exception = ex });
+            }
+        }
+
+        private async Task<bool> ProcessFile(string downloadLocation, string documentId, string fileFormat, string downloadType)
+        {
+            var recordingDownloadUri = new Uri(downloadLocation);
+            var response = DownloadExtentions.DownloadStreamingAsync(callingServerClient, recordingDownloadUri);
```

Comment:
`DownloadExtentions.DownloadStreamingAsync` was undefined / could not be found when I went to build.
Did you forget to include a file / was this a holdover?